internal/provider/resource_gitlab_project_membership.go

```diff
@@ -153,7 +163,9 @@ func resourceGitlabProjectMembershipSetToState(d *schema.ResourceData, projectMe
 	d.Set("project_id", projectId)
 	d.Set("user_id", projectMember.ID)
 	d.Set("access_level", accessLevelValueToName[projectMember.AccessLevel])
-
+	if projectMember.ExpiresAt != nil {
+		d.Set("expires_at", projectMember.ExpiresAt.String())
+	}
```

Comment:
If `projectMember.ExpiresAt` comes back as `nil` we have to clear the `expires_at` state field:

```suggestion
	if projectMember.ExpiresAt != nil {
		d.Set("expires_at", projectMember.ExpiresAt.String())
	} else {
	         d.Set("expires_at", "")
	}
```

Reply:
thx, I've added your suggestion